src/page-viewer/styles.css

```diff
@@ -0,0 +1,6 @@
+.root {
+    float: right;
+    padding: 4px 4px 4px 12px;
```

Comment:
Merging the `props.className` with its own `className` looks like a logical solution. I think however that in this particular case the component does not really need reasonable defaults to be set, as a normal unstyled link with underline is a reasonable default too (of course an underline would look weird on the unicode symbol we use, but that symbol is passed as `children`, so its style is not up to the component).
In short: just put the whole style in overview.